error[E0463]: can't find crate for `rustc_literal_escaper` which `proc_macro` depends on
   --> ../../third_party/rust/chromium_crates_io/vendor/proc-macro2-v1/src/lib.rs:139:1
    |
139 | extern crate proc_macro;
    | ^^^^^^^^^^^^^^^^^^^^^^^^ can't find crate
    |
    = help: maybe you need to install the missing components with: `rustup component add rust-src rustc-dev llvm-tools-preview`

error: aborting due to 1 previous error

For more information about this error, try `rustc --explain E0463`.
--- a/build/rust/std/BUILD.gn
+++ b/build/rust/std/BUILD.gn
@@ -40,6 +40,7 @@
     "panic_abort",
     "panic_unwind",
     "rustc_demangle",
+    "rustc_literal_escaper",
     "std_detect",
     "test",
     "unicode_width",
